mage_ai/data_preparation/models/variable.py

```diff
@@ -399,10 +399,12 @@ def __write_parquet(self, data: pd.DataFrame) -> None:
                         # df_output[c] = series_non_null.astype(str)
                         pass
 
-                if coltype.__module__ == np.__name__:
-                    column_types[c] = type(series_non_null.iloc[0].item()).__name__
-                else:
+                colNotNumpyType = coltype.__module__ != np.__name__
```

Comment:
could you use snake case for param name?

Reply:
@wangxiaoyou1993 so sorry for the mistake, accidentally brought my habit from other projects 🙏 . It has been changed to snakecase